socorro/schemas/validate_processed_crash.py

```diff
@@ -113,8 +117,12 @@ def match_key(schema_key, document_key):
 @click.argument("crashdir")
 @click.pass_context
 def validate_and_test(ctx, crashdir):
-    jsonschema.Draft4Validator.check_schema(PROCESSED_CRASH_SCHEMA)
-    click.echo("processed crash schema is a valid JSON schema.")
+    socorro_data_schema = get_file_content("socorro-data-1-0-0.schema.yaml")
```

Comment:
This fixes `validate_processed_crash.py` so it can (sort of) be used to validate that the schemas themselves are valid. I say "sort of" because you still have to provide a bunch of processed crash examples and sometimes I pass it an empty directory which allows it to "work", but it's wonky.